rest-server/engines/src/main/scala/com/actionml/engines/urnavhinting/URNavHintingDataset.scala

```diff
@@ -124,7 +124,7 @@ class URNavHintingDataset(engineId: String, val store: Store) extends Dataset[UR
           case "$delete" =>
             if (event.entityType == "user") {
               // this will only delete a user's data
-              itemsDao.removeOne(filter = ("entityId", event.entityId)) // removeOne all events by a user
+              //itemsDao.removeOne(filter = ("entityId", event.entityId)) // removeOne all events by a user
             } // ignore any other reserved event types, they will be caught by the Algorithm if at all
```

Comment:
this code is wrong. But in any case will be executed in a Job because it might take longer than the REST timeout allows.